x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -145,6 +146,22 @@ export class Explorer extends React.Component {
   state = { filterIconTriggeredQuery: undefined, language: DEFAULT_QUERY_LANG };
   htmlIdGen = htmlIdGenerator();
 
+  componentDidMount() {
+    const { invalidTimeRangeError } = this.props;
+    if (invalidTimeRangeError) {
+      const toastNotifications = getToastNotifications();
+      toastNotifications.addWarning(
+        i18n.translate('xpack.ml.explorer.invalidTimeRangeInUrlCallout', {
+          defaultMessage:
+            'The time filter changed to the full range for this job due to invalid default time filter. Please check the advanced settings for {field}.',
```

Comment:
Nit - suggest changing to `The time filter was changed to the full range for this job due to an invalid default time filter. Please check the advanced settings for {field}.`

Reply:
> Please check the advanced settings. 

Another nit: The style guidelines recommend omitting "Please".  I'd suggest "Check the advanced settings...".

Reply:
Updated it to `The time filter was changed to the full range due to an invalid default time filter. Check the advanced settings for {field}.` here https://github.com/elastic/kibana/pull/76347/commits/1118b3e08345110bec102f86bfbd3bf46cce969c